telemetry/packages/public-app/components/gridDisplay.tsx

```diff
@@ -0,0 +1,126 @@
+import { Card, Title, Text, Grid, Badge } from '@tremor/react';
+import { VelocityGraph } from './velocity-graph';
+
+
+import { useState } from 'react';
+import { MeasurementChart } from './measurementsGraph';
+import { DisplacementChart } from './DisplacementChart';
+
+import Image from 'next/image';
+import { useEffect } from 'react';
+import { DigitalTimer } from './timer';
+import Switch from 'react-switch';
+import LevitationHeight from './levitationHeight';
+import { SocialIcons } from './socialIcons';
+
+const CARDS = {
+  VELOCITY: <VelocityGraph />,
+  ACCELERATION: <DisplacementChart />,
+  LEVITATION: <LevitationHeight />,
+  
+};
+
+type Card = keyof typeof CARDS;
+
+export const ColorInverter = () => {
+  if (localStorage.getItem('color-theme')) {
+    if (localStorage.getItem('color-theme') === 'light') {
+      localStorage.setItem('color-theme', 'dark');
+      document.documentElement.classList.remove('light');
+      document.documentElement.classList.add('dark');
+    } else if (localStorage.getItem('color-theme') === 'dark') {
+      localStorage.setItem('color-theme', 'light');
+      document.documentElement.classList.add('light');
+      document.documentElement.classList.remove('dark');
+    }
+  } else {
+    localStorage.setItem('color-theme', 'dark');
+    document.documentElement.classList.add('dark');
+  }
+};
+
+export default function GridDsiplay() {
+  const [selected, setSelected] = useState<Card>('VELOCITY');
+  const [clicked, setClicked] = useState(false);
+
+  const switched = () => {
+    ColorInverter();
+    !clicked && localStorage.getItem('color-theme') === 'dark'
+      ? setClicked(true)
+      : setClicked(false);
+  };
+
+  return (
+    <main className="w-[360px] md:w-[720px] lg:w-[1000px] mx-auto py-12">
+      <div className="nav p-2 mb-3">
+        <div className="heading  p-2">
+          {' '}
+          <div>
+            <Image
+              alt="Hyped logo"
+              src={
+                clicked === false ? '/hypedLogoLight.png' : '/hypedLogoDark.png'
```

Comment:
not an ideal way to deal with theme state, could use a global context instead. Something like this: https://javascript.plainenglish.io/how-to-implement-dark-light-themes-in-a-next-js-app-using-context-hook-tailwindcss-336558dd4579#:~:text=The%20context%20for%20toggling%20theme